pkg/providers/subnet/subnet.go

```diff
@@ -82,6 +83,12 @@ func (p *Provider) LivenessProbe(req *http.Request) error {
 	return nil
 }
 
+func (p *Provider) ResetCache() {
```

Comment:
Why does subnet provider need to "Reset" but the security group provider doesn't need to?

Reply:
They both have reset function that was a duplicate.